parm/archive/enkf.yaml.j2

```diff
@@ -70,6 +70,14 @@ enkf:
         - "{{ COMIN_ATMOS_ANALYSIS_ENSSTAT | relpath(ROTDIR) }}/{{ head }}{{ file }}"
         {% endfor %}
 
+        {% if DO_JEDISNOWDA %}
```

Comment:
```suggestion
        {% if "{{ DO_JEDISNOWDA }}" %}
```

Reply:
This seems strange, are you sure this doesn't work without this change?

Reply:
I tested that the run failed due to the undefined `DO_JEDISNOWDA`.

Reply:
Ok I'll look into it, none of the similar variables are in quotes anywhere so maybe it needs added someplace. I'm building a fresh version now and will dig into it. Thank you @jiaruidong2017 for testing these changes.

Reply:
See the logfile below for your reference:

`/scratch1/NCEPDEV/climate/Jiarui.Dong/ptmp/COMROOT/C384C192snowda_2dvar/logs/2021122118/enkfgdas_earc00.log`

Reply:
@jiaruidong2017 , I reproduced the error you see,  but @CoryMartin-NOAA 's point is valid.  The C96C48_hybatmaerosnowDA gdas_arch job runs to completion.  `gdas.yaml.j2` has
```
            {% if DO_JEDISNOWDA %}
        - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ head }}snowstat.tgz"
            {% endif %}
```
This works.  Seems something else is going wrong in the earc00 job.

Reply:
I haven't tested it, but I think the solution to this issue is there needs to be a `DO_JEDISNOWDA` key added to this section https://github.com/CoryMartin-NOAA/global-workflow/blob/feature/snow-da-refactor/scripts/exgdas_enkf_earc.py#L20C1-L31C35

Reply:
testing it now, Travis, I just pushed the commit, we came to the same conclusion concurrently

Reply:
@TravisElless-NOAA , you are correct.  I added DO_JEDISNOWDA to `scripts/exgdas_enkf_earc.py`
```
diff --git a/scripts/exgdas_enkf_earc.py b/scripts/exgdas_enkf_earc.py
index 467cfa88..1b706a12 100755
--- a/scripts/exgdas_enkf_earc.py
+++ b/scripts/exgdas_enkf_earc.py
@@ -22,7 +22,7 @@ def main():
             'PSLOT', 'HPSSARCH', 'LOCALARCH', 'ROTDIR', 'PARMgfs',
             'ARCDIR', 'SDATE', 'MODE', 'ENSGRP', 'NMEM_EARCGRP',
             'NMEM_ENS', 'DO_CALC_INCREMENT_ENKF_GFS', 'DO_JEDIATMENS',
-            'lobsdiag_forenkf', 'FHMIN_ENKF', 'FHMAX_ENKF_GFS',
+            'DO_JEDISNOWDA', 'lobsdiag_forenkf', 'FHMIN_ENKF', 'FHMAX_ENKF_GFS',
             'FHOUT_ENKF_GFS', 'FHMAX_ENKF', 'FHOUT_ENKF', 'ENKF_SPREAD',
             'restart_interval_enkfgdas', 'restart_interval_enkfgfs',
             'DOHYBVAR', 'DOIAU_ENKF', 'IAU_OFFSET', 'DOIAU', 'DO_CA',
```
A rewind and reboot of 20211221 00Z enkfgdas_earc00 ran past the DO_JEDISNOWDA problem.  

The job failed with
```
FileNotFoundError: FATAL ERROR: Required file, directory, or glob enkfgdas.20211221/00/ensstat/analysis/snow/enkfgdas.t00z.snowstat.tgz not found!
+ JGDAS_ENKF_ARCHIVE[1]: postamble JGDAS_ENKF_ARCHIVE 1734625224 1
```
Looks like I need to back up and run additional jobs to clear out this new error.